tests/paste_events/_helpers/pasting.js

```diff
@@ -0,0 +1,72 @@
+/* exported assertPasteEvent, pasteFiles */
+
+'use strict';
+
+function assertPasteEvent( editor, eventData, expected, message, async ) {
```

Comment:
so this file is actually a helper from the clipboard plugin.  I tried symlinking it, but I couldn't get it to work, so I just copy / pasted